Include ApplicationServices instead of CoreGraphics.
CoreGraphics didn't become a top-level framework until OS X 10.8.
https://gitlab.gnome.org/GNOME/gimp/issues/2113
--- app/widgets/gimpwidgets-utils.c.orig	2018-08-19 11:47:18.000000000 -0500
+++ app/widgets/gimpwidgets-utils.c	2018-08-26 22:39:51.000000000 -0500
@@ -34,7 +34,7 @@
 #endif
 
 #ifdef PLATFORM_OSX
-#include <CoreGraphics/CoreGraphics.h>
+#include <ApplicationServices/ApplicationServices.h>
 #endif
 
 #include "libgimpbase/gimpbase.h"
